docs/modules/operation/pages/deep-dive/admin/configuration/scv-configuration.adoc

```diff
@@ -7,6 +7,17 @@ It stores them in the Java keystore (JKS) format.
 
 The secure credentials vault is enabled by default.
 
+== Configure keystore type and file name
+By default, the Secure Credentials Vault in OpenNMS uses a `JCEKS` keystore with the file name `scv.jce`.
+OpenNMS supports two keystore types: `JCEKS` and `PKCS12`. You can customize keystore type by setting the appropriate properties in:
+`$\{OPENNMS_HOME}/etc/opennms.properties.d/scv.properties.`
++
+[source, xml]
+----
+org.opennms.features.scv.jceks.keystore.type=PKCS12
+----
+NOTE: If you change the keystore type to `PKCS12`, be sure to update the file extension to match the format (e.g., use `.pk12` instead of `.jce`).
```

Comment:
I guess this note is only intended for people who need to convert from JCEKS to PKCS12 format.  So update this note that reflects that.

This note may be required in whatsnew.doc as well.